examples/crd-client/main.go

```diff
@@ -66,8 +83,14 @@ func main() {
 		logger.WithError(err).Fatal("Could not create the agones api clientset")
 	}
 
+	gsName := "helm-test-server-"
```

Comment:
Minor: I see that this variable is used only in 91 line, so let's make this code a bit cleaner and move this value either to gs creation step or make it constant and move to the top of the file.